pkg/vsphere/toolbox/backdoor_test.go

```diff
@@ -30,13 +30,8 @@ func TestBackdoorChannel(t *testing.T) {
 	}
 
 	for _, f := range funcs {
-		err := f()
-
-		if err != nil {
-			if err == ErrNotVirtualWorld {
```

Comment:
Did this check not work anymore?  Only wanted to skip the test on this error.  If we can't do that check, adding err.Error() as a param to SkipNow would be good.